.env/pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /Library/Developer/CommandLineTools/usr/bin
```

Comment:
Should this file be in version control? I'm not familiar with it. But it does look like a py3 thing.

Reply:
nope. my .env folder should be ignored...